flask_resources/parsers/schema.py

```diff
@@ -67,6 +69,20 @@ def get_aggs(self, obj_list):
         if not aggs:
             return missing
         return aggs
+    
+    def get_links(self, obj_list):
```

Comment:
did you check if we have unit tests?
you could also check if it works integrated with invenio-rdm-records and invenio-communities modules (and their tests) if not done already. 
I am suggesting this because maybe some resources already have their own implementations of get_links and get_sorting_options and I would like to make sure they still work OK.